### ISSUE TITLE
Overlay icon like from LSE doesn't seem to work on a file

### ISSUE THREAD

**Issue by gitgt** [doublecmd/doublecmd#762]
### Version

Double Commander Version: 1.0.9 Platform: x86_64-Win64-win32/win64 OS version: Windows 10 21H2 x86_64

### Description

In Windows I'm using Link Shell Extension from https://schinagl.priv.at/nt/hardlinkshellext/linkshellextension.html.
When it makes for example Hardlink, icon on a file is changed, seen in Windows Explorer, called "Overlay Icons".
I don't see that icon in DC, I still see an original filetype icon. 
Please see, thanks.

**Comment by Skif-off** [comment]
[Configuration > Options... > Icons](https://doublecmd.github.io/doc/en/configuration.html#ConfigIcons)> `All associated + EXE/LNK (slow)` + `Show overlay icons, e.g. for links`?

**Comment by gitgt** [comment]
Yes. Thanks. I searched in bugs, didn't remember to look in options. 
And https://doublecmd.github.io/doc/en/index.html doesn't have search.
